ballerina/Dependencies.toml

```diff
@@ -5,7 +5,7 @@
 
 [ballerina]
 dependencies-toml-version = "2"
-distribution-version = "2201.8.0"
+distribution-version = "2201.8.8-20240816-210100-3e1bef6f"
```

Comment:
Any reason to use a timestamped version?

Reply:
Fixed with [8cf9e162d55fc6d0a521ce7c403e7ac3ab811044](https://github.com/ballerina-platform/module-ballerina-mqtt/pull/204/commits/8cf9e162d55fc6d0a521ce7c403e7ac3ab811044)